internify/src/components/pages/CreateJobPosting/ContactDetails.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import Container from "@material-ui/core/Container";
+import { ButtonFilled } from "../../atoms/Button";
+import {
+    Stepper,
+    InputFormContactDetails
+  } from "../../molecules/index";
+import "./styles/ContactDetails.css";
+
+const ContactDetails = () => {
+    return (
+        <div className="contact_details">
+            <Container maxWidth="md" className={"container"}>
```

Comment:
can you add the styling as follows to the container element tag so all Job posting pages have same container padding: 

style={{ padding: "0 10em" }}

Reply:
Do I just name it .container? Since the className is {"container"}, and I'm not sure if that's different.